.github/workflows/awstest.yml

```diff
@@ -26,5 +28,10 @@ jobs:
           AWS_ACCESS_KEY_ID: ${{secrets.AWS_KEY_ID}}
           AWS_SECRET_ACCESS_KEY: ${{secrets.AWS_KEY_SECRET}}
           TOWER_ACCESS_TOKEN: ${{secrets.TOWER_ACCESS_TOKEN}}
```

Comment:
```suggestion
          AWS_ACCESS_KEY_ID: ${{secrets.AWS_ACCESS_KEY_ID}}
          AWS_SECRET_ACCESS_KEY: ${{secrets.AWS_SECRET_ACCESS_KEY}}
          TOWER_ACCESS_TOKEN: ${{secrets.AWS_TOWER_TOKEN}}
          AWS_JOB_DEFINITION: ${{ secrets.AWS_JOB_DEFINITION }}
          AWS_JOB_QUEUE: ${{ secrets.AWS_JOB_QUEUE }}
          AWS_S3_BUCKET: ${{ secrets.AWS_S3_BUCKET }}
```

Reply:
Will be fixed in https://github.com/nf-core/viralrecon/pull/118